analysis.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"strings"
+)
+
+type Info struct {
+	Name      string
+	Namespace string
+	Kind      string
+	Count     int
+	Message   string
+}
+
+type Result struct {
+	CheckName     string `json:"checkName" yaml:"checkName"`
+	Status        int    `json:"status" yaml:"status"`
+	StatusMessage string `json:"statusMessage" yaml:"statusMessage"`
+	Info          []Info `json:"info" yaml:"info"`
+}
+
+type Events struct {
+	Items []struct {
+		Kind           string `json:"kind"`
+		InvolvedObject struct {
+			Namespace string `json:"namespace"`
+			Name      string `json:"name"`
+		} `json:"involvedObject"`
+		Reason  string `json:"reason"`
+		Message string `json:"message"`
+		Count   int    `json:"count"`
+	} `json:"items"`
+}
+
+type DeploymentConfigs struct {
```

Comment:
These types here should match (be a subset of) an specific API version from OpenShift / Kubernetes.
As I mentioned earlier, I'd strongly suggest these to be taken from their original definitions, unneeded parts trimmed, and commentary linking back to where it was taken from (including version information).